plugins/inputs/bond/README.md

```diff
@@ -0,0 +1,93 @@
+# Bond Input Plugin
+
+The Bond Input plugin collects bond interface status, bond's slaves interfaces
+status and failures count of bond's slaves interfaces.
+The plugin collects these metrics from `/proc/net/bonding/*` files.
+
+### Configuration:
+
+```toml
+[[inputs.bond]]
+  ## Sets 'proc' directory path
+  ## If not specified, then default is /proc
+  # host_proc = "/proc"
+
+  ## Sets 'bonding' directory relative path
+  ## If not specified, then default is /net/bonding
+  # bond_path = "/net/bonding"
```

Comment:
Would this ever not be "/net/bonding"?

Reply:
Seems no, would you like me to hardcode this parameter?

Reply:
Yeah, will make the interface simpler which is always a good thing.

Reply:
done